cmd/kubeadm/app/phases/addons/dns/versions.go

```diff
@@ -22,13 +22,17 @@ import (
 
 const (
 	kubeDNSv170AndAboveVersion = "1.14.5"
+	kubeDNSv190AndAboveVersion = "1.14.6"
 )
 
 // GetKubeDNSVersion returns the right kube-dns version for a specific k8s version
 func GetKubeDNSVersion(kubeVersion *version.Version) string {
 	// v1.7.0+ uses 1.14.5, just return that here
 	// In the future when the kube-dns version is bumped at HEAD; add conditional logic to return the right versions
 	// Also, the version might be bumped for different k8s releases on the same branch
+	if kubeVersion.AtLeast(version.MustParseGeneric("1.9.0")) {
```

Comment:
I would use `if kubeVersion.Minor() >= 9` so that alpha/beta/rc versions also will match this new dns version